src/main/resources/templates/timeentries/_navigation.html

```diff
@@ -106,19 +107,25 @@
                     ></div>
                   </div>
                 </details>
-                <form action="#" th:action="@{/timeclock/stop}" method="post">
+                <form
+                  action="#"
+                  th:action="@{/timeclock/stop}"
+                  method="post"
+                  class="flex items-center"
+                >
                   <button
                     type="submit"
-                    class="bg-red-700 text-white pl-3 pr-4 py-1 text-lg font-medium rounded-full flex items-center hover:bg-red-600 transition-colors"
+                    class="bg-red-700 text-white py-1 pl-2 pr-3 text-lg font-medium rounded-full flex items-center gap-1.5 hover:bg-red-600 transition-colors"
                   >
                     <svg
-                      th:replace="icons/stop::svg(className='w-6 h-6 mr-1.5')"
+                      th:replace="icons/stop::svg(className='text-red-100 w-6 h-6')"
                     />
                     <span th:text="#{timeclock.stop}"> Stop </span>
                   </button>
                 </form>
               </div>
-              <div class="ml-8 shrink-0 relative">
+              <div th:replace="launchpad/launchpad::navigation-item"></div>
```

Comment:
Das ist ja der Part für `/timeentries`. Wodurch wird das in den Reports eingebunden?

Reply:
Das ist hysterisch gewachsen. `_layout` wird "überall" verwendet als Basis.

Reply:
ups okay, übersehen. vllt rausziehen?